.github/workflows/update-website.yml

```
@@ -0,0 +1,50 @@
+name: Update latest generator documentation in the website
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'docs/*.md'
+
+jobs:
+  Make-PR:
+    name: Make PR on website repository with updated latest generator documentation
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+    steps:
+      - name: Checkout Current repository
+        uses: actions/checkout@v2
+        with:
+          path: generator
+      - name: Checkout Another repository
+        uses: actions/checkout@v2
+        with:
+          repository: asyncapi/website
+          path: website
+          token: ${{ env.GITHUB_TOKEN }}
+      - name: Config git
+        run: |
+          git config --global user.name asyncapi-bot
+          git config --global user.email info@asyncapi.io
+      - name: Create branch
+        working-directory: ./website
+        run: |
+          git checkout -b update-generator-docs-${{ github.sha }}
+      - name: Copy generator folder from Current Repo to Another
+        working-directory: ./website
+        run: |
+          mkdir -p ./pages/docs/tools/generator
+          cp -R ../generator/docs ./pages/docs/tools/generator
+          printf "\`\`\`\ntitle: Generator\nweight: 3\n\`\`\`" > _section.md
```

```suggestion
          mkdir -p ./pages/docs/tools/generator
          rm ../generator/docs/README.md
          cp -R ../generator/docs ./pages/docs/tools/generator
          printf "\`\`\`\ntitle: Generator\nweight: 3\n\`\`\`" > _section.md
```

README will be removed only in a clone done in workflow, not from the upstream